lighthouse-core/report/html/renderer/util.js

```diff
@@ -72,6 +72,21 @@ class Util {
       }
     }
 
+    if (clone.stackPacks) {
+      for (const audit of Object.values(clone.audits)) {
+        audit.stackPacks = {};
```

Comment:
Right now we'll be creating this object on each audit even if there are no extra descriptions defined. Seems like we should only create it if there exist descriptions.

(See also keyed object comment.)

Reply:
👍